src/helpers/clearToken.js

```diff
@@ -0,0 +1,8 @@
+const clearToken = (token) => {
+  if (token) {
+    return window.localStorage.removeItem('token');
+  }
+  return window.localStorage.clear();
```

Comment:
@NiyoEric Why do we need to clear everything in local storage?

Reply:
I think there is a way to improve this function. You would improve to validate these conditions:

1. If the token exists in local storage.
2. If the token exists then remove it.
3. and also there is no need to clear all the items in local storage.